src/main/java/com/epam/test_generator/entities/api/UserTrait.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.test_generator.entities.api;
+
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+import com.epam.test_generator.entities.Role;
+import com.epam.test_generator.entities.User;
+
+public interface UserTrait {
+
+    User getUser();
```

Comment:
@AleksandrKugushev suggests that is an overabundant method, so instead of this declare methods that you need and use theirs explicitly.